app/code/Magento/CatalogSearch/Test/Unit/Model/Search/FilterMapper/TermDropdownStrategyTest.php

```diff
@@ -27,89 +27,53 @@ class TermDropdownStrategyTest extends \PHPUnit\Framework\TestCase
     private $eavConfig;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
-     */
-    private $storeManager;
-
-    /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var TermDropdownStrategy
      */
-    private $scopeConfig;
+    private $model;
```

Comment:
Try to avoid not clear names like as `model`

Reply:
Fixed.